src/modules/ROOT/pages/changelog/202303/06.json

```diff
@@ -5,18 +5,18 @@
   "added": [
     {
       "what": [
-        "splice function in workflows that can change the contents of a collection of elements by removing or replacing existing elements and/or adding new elements in place."
+        "splice function in workflows that can change the contents of an array by removing or replacing existing elements of different types like strings, numbers, etc. and/or adding new elements in place."
       ],
       "why": [
-        "to provide the option to easily add/remove elements from a collection."
+        "to provide the option to easily add/remove elements from an array."
```

Comment:
List other types as well; above you mention strings and numbers; should we do the same here?